### ISSUE TITLE
Displaced gridster-item on fixed grid type

### ISSUE THREAD

**Issue by klemenoslaj** [tiberiuzuld/angular-gridster2#305]
When `maxCols` is specified with fixed gridType and `gridster-item` exceeds the `maxCols` amount, that item gets pushed down.

# Screenshot
![screen shot 2018-04-16 at 16 38 09](https://user-images.githubusercontent.com/7548247/38815615-a3acaa18-4194-11e8-83fe-7f9d9b1ace45.png)

# Settings to reproduce

### Template
```
<gridster [options]="options">
  <gridster-item [item]="item" *ngFor="let item of dashboard" style="background: yellow;"></gridster-item>
</gridster>
```

### GridsterConfig
```
this.options = {
    gridType: GridType.Fixed,
    maxCols: 4,
};
```

### GridsterItem
```
[{
    cols: 5,
    rows: 4,
    y: 0,
    x: 0
}];
```

**Comment by tiberiuzuld** [comment]
Hi @klemenoslaj ,
This happens in last version 5.2.1 ? If yes will take a closer look this week.

**Comment by klemenoslaj** [comment]
Hi @tiberiuzuld,

Yes to verify I cloned your repo and hardcoded the reproducible setup, so I was sure it is a clean place to test.

**Comment by tiberiuzuld** [comment]
Fixed in v5.3.0
